CRM/Event/Form/Search.php

```diff
@@ -79,6 +79,9 @@ public function getDefaultEntity() {
    * @throws \CiviCRM_API3_Exception
    */
   public function preProcess() {
+    // SearchFormName is deprecated & to be removed - the replacement is for the task to
+    // call $this->form->getFormValues()
```

Comment:
I think all these new comment blocks have a typo. Shouldn't it be
```suggestion
    // call $this->form->getSearchFormValues()
```

Reply:
oh yeah - I renamed the fn - will fix

Reply:
@colemanw fixed